playbooks/common/openshift-cluster/disable_excluder.yml

```diff
@@ -3,9 +3,29 @@
   hosts: l_oo_all_hosts
   gather_facts: no
   tasks:
+
+  # During installation the excluders are installed with present state.
+  # So no pre-validation check here as the excluders are either to be installed (present = latest)
+  # or they are not going to be updated if already installed
+
+  # Install any excluder that is enabled
+  - include_role:
+      name: openshift_excluder
+      tasks_from: init
+  - include_role:
+      name: openshift_excluder
+      tasks_from: install
```

Comment:
Having to use two include_role tasks seems a bit fragile. Would it make sense to conditionally include `install` from within `init` in the role or have `install` just include `init` prior to doing other tasks?